WSRequestHandler.cpp

```diff
@@ -295,6 +297,41 @@ void WSRequestHandler::HandleAuthenticate(WSRequestHandler* req) {
     }
 }
 
+/************************************************************************************************************
+* Heatbeat update message is emitted every 2 seconds, when enabled with this request.                       *
+* When the Heartbeat is enabled it always sends a `pulse` to indicate that the host obs is alive.           *
+* Read comment from 'void WSEvents::Heartbeat()' for the total picture.                                     *
+*                                                                                                           *
+* @param {string} `mode` Value options: `off`, `on`.                                                        *
```

Comment:
When a string value only has two possible values, it's better to refactor it as a boolean. So here, `mode` (string parameter) should become `enabled` (boolean parameter).

Reply:
For your consideration my thoughts... From a product design point of view, when the key is called "mode" it is better to use string, as you can grow the number of values in the future, and the interface maintains backwards compatible. If truly only a binary value is intended for the long run, than it might be better not to use the key word "mode", but use "enable" or "active" instead. Please let me know how you like to see it and I will make the update.